client.go

```diff
@@ -962,3 +962,27 @@ func (client *gocloak) DeleteUserFromGroup(token string, realm string, userID st
 
 	return checkForError(resp, err)
 }
+
+// GetEvents get all events in a realm
+func (client *gocloak) GetEvents(token string, realm string, params *GetEventsParams) ([]EventRepresentation, error) {
+	var result []EventRepresentation
+	queryParams, err := GetQueryParams(params)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := client.getRequestWithBearerAuth(token).
+		SetResult(&result).
+		SetQueryParams(queryParams).
+		Get(client.getAdminRealmURL(realm, "events"))
```

Comment:
One more, please take a look at the latest (6.0) documentation: https://www.keycloak.org/docs-api/6.0/rest-api/index.html

The event urls were changed from `GET /admin/realms/{realm}/events` to `GET /{realm}/events`
I think we need to support both or the latest.